### ISSUE TITLE
 RuntimeException (400) Template "external.html.twig" is not defined.

### ISSUE THREAD

**Issue by chross** [robbinfellow/haywire-grav#12]
When trying to clone the Github page to insert another external link, I get the following error:
```
RuntimeException (400)
Template "external.html.twig" is not defined.
```

I'm using haywire 0.3.3 and Grav v1.4.0-rc.1 with Admin v1.7.0-rc.2

UPDATE:
Ok, so it seem that some settings did this:
I made the page non-visible and then entered its direct path into the browser bar.

**Comment by robbinfellow** [comment]
Hi,
I'm trying to replicate this issue but I'm unable to encounter any errors. Duplicating the page both manually and from the admin panel works fine.

Can you provide *steps to reproduce the issue* for me please?

**Comment by sebastienhouzet** [comment]
Hello

For replicate this error, use defaut theme Antimatter  v2.1.2.
Create a new page and select Page Template : External.

Sébastien

**Comment by robbinfellow** [comment]
@sebastienhouzet haywire theme has nothing to do with the default `antimatter` theme of grav. 

The external template is something I used when creating this theme, there's nothing I can do to change the behavior of `antimatter` theme.

I hope I understood this correctly, otherwise please let me know 👍 

Thanks for using haywire 🙌

**Comment by sebastienhouzet** [comment]
@robbinfellow Yes, you have understand :)

I have just write this, beacause with the default theme, i have the same error and it is possible is a bug on Grav and not on your Theme.

Sébastien

**Comment by chross** [comment]
Antimatter has the same "problem". I'd say it is a usage out of the intended scope. Best is to use page redirect, at least that works for me.

**Comment by robbinfellow** [comment]
@sebastienhouzet Ah, I see! I didn't fully understand your input but now I do. Indeed it looks like it has something to do with grav itself since the same error occurs when using the default `antimatter` theme.

@chross I see, then it's probably like @sebastienhouzet mentioned – an issue with grav itself and not the theme.

If you had a working solution to circumvent this, feel free to submit a PR, I'll look into this issue as well, and for ways to prevent it from happening in future releases of haywire.

**Comment by robbinfellow** [comment]
@sebastienhouzet @chross 

I have [committed a fix](https://github.com/robbinfellow/haywire-grav/commit/53540039bf60ae6a4322bd16cec28565b927bcc8) for this issue now.

#### What will happen now when trying to visit a page using the external page template is:

Haywire will try to redirect to the given `external_url` when using the external page template.

If the `external_url` is not set (which basically shouldn't be possible since it's a **_required_** field) and a visitor somehow would be able to access a page using the external template I've added an [external page template](https://github.com/robbinfellow/haywire-grav/blob/develop/templates/external.html.twig) as a fallback to simply display an error that the page is not available.

This will be available soon, in `0.3.4` 🚀

**Comment by robbinfellow** [comment]
@sebastienhouzet @chross 

Fixed in `0.3.4`. 

Go ahead and update:

`$ bin/gpm update haywire`
